docs/src/use/command-line-interface.md

```diff
@@ -86,7 +86,7 @@ Fix Problems:
 
 Ignore Files:
   --no-ignore                     Disable use of ignore files and patterns
-  --ignore-pattern [String]       Pattern of files to ignore (in addition to those in .eslintignore)
+  --ignore-pattern [String]       Patterns of files to ignore
```

Comment:
This option also works with legacy config, so this info might still be useful for those users depending upon the config.

Reply:
I've updated the texts as suggested by @nzakas. Does the new wording look good to you?